### ISSUE TITLE
Broken http4s dependency version in v2.0.0

### ISSUE THREAD

**Issue by nigredo-tori** [pac4j/http4s-pac4j#12]
With SBT 1.4.6, and this `build.sbt`:
```scala
libraryDependencies += "org.pac4j" %% "http4s-pac4j" % "2.0.0"
```
, `sbt update` produces the following error:
```
[error] (update) sbt.librarymanagement.ResolveException: Error downloading org.http4s:http4s-circe_2.12:[0.21.6,0.21.7),[0.21.60,0.21.70),[0.21.600,0.21.700),[0.21.6000,0.21.7000),[0.21.60000,0.21.70000)
[error]   No version found for [0.21.60000,0.21.70000) in 
[error]   No version found for [0.21.60000,0.21.70000) in 
[error]   not found: /home/dmitry/.ivy2/local/org.http4s/http4s-circe_2.12
[error]   No version found for [0.21.60000,0.21.70000) in https://repo1.maven.org/maven2/org/http4s/http4s-circe_2.12/maven-metadata.xml
[error] Error downloading org.http4s:http4s-dsl_2.12:[0.21.6,0.21.7),[0.21.60,0.21.70),[0.21.600,0.21.700),[0.21.6000,0.21.7000),[0.21.60000,0.21.70000)
[error]   No version found for [0.21.60000,0.21.70000) in 
[error]   No version found for [0.21.60000,0.21.70000) in 
[error]   not found: /home/dmitry/.ivy2/local/org.http4s/http4s-dsl_2.12
[error]   No version found for [0.21.60000,0.21.70000) in https://repo1.maven.org/maven2/org/http4s/http4s-dsl_2.12/maven-metadata.xml
[error] Error downloading org.http4s:http4s-server_2.12:[0.21.6,0.21.7),[0.21.60,0.21.70),[0.21.600,0.21.700),[0.21.6000,0.21.7000),[0.21.60000,0.21.70000)
[error]   No version found for [0.21.60000,0.21.70000) in 
[error]   No version found for [0.21.60000,0.21.70000) in 
[error]   not found: /home/dmitry/.ivy2/local/org.http4s/http4s-server_2.12
[error]   No version found for [0.21.60000,0.21.70000) in https://repo1.maven.org/maven2/org/http4s/http4s-server_2.12/maven-metadata.xml
```
This is probably caused by this line:

https://github.com/pac4j/http4s-pac4j/blob/365ffd0be9fa525bd23eaea9bdd32f3f41f77875/build.sbt#L6

It should be `0.21.6` rather than `0.21.6+`.

**Comment by leleuj** [comment]
I'm no sbt specialist, but I'm surprised. Can't you force the version of `http4s` to avoid this issue?

**Comment by nigredo-tori** [comment]
[I can](https://www.scala-sbt.org/1.x/docs/Library-Management.html#Exclude+Transitive+Dependencies), but I shouldn't have to. :smile: It's usually assumed that adding a library to `libraryDependencies` would be enough.

BTW, I'm not sure `0.21.6+` is a valid Ivy revision in the first place, which may be the reason for this weird range expansion (`[0.21.60,0.21.70)` and so on). There might be a way to specify the desired version range (whatever that  is) so that downstream SBT would be OK with it. But I don't think digging into this would be worth the trouble.

**Comment by leleuj** [comment]
OK. I fixed the version, I'll cut a v2.0.1 release later on.

**Comment by leleuj** [comment]
The release v2.0.1 should show up in the Maven central repository within a few hours...
